tests/conftest.py

```diff
@@ -1,113 +1,56 @@
+import sys
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).parent))
+
 import pytest
 import shutil
-from pathlib import Path
 import yaml
-import pyaudio
+import set_up_sparrow as spf
 
 HOME = None
 DATA = None
 OUTPUT = None
-
-
-# README: the below will later land in setup.py...
-def read_yaml(path: str):
-    """
-        read_yaml Read the yaml basic config file for iSparrow from path.
-                It contains the install directory, data directory and other things used
-                by iSparrow internally.
-
-        Args:
-            path (str): Path to the yaml base config.
-
-        Returns:
-            dict: read base config file.
-        """
-
-    if Path(path).exists() is False:
-        raise FileNotFoundError(f"The folder {path} does not exist")
-
-    with open(Path(path)) as file:
-        base_cfg = yaml.safe_load(file)
-
-    return base_cfg
-
-
-def make_directories(base_cfg_dirs: dict):
-    """
-    make_directories Make all the directories for sparrow.
-
-
-    Args:
-        base_cfg_dirs (dict): Dictionary containing paths for the main install ("home"),
-        the directory where models are stored ("models"), the one where data may be stored ("data")
-        and the "output" directory to store inference results and potentially other data in ("output")
-
-    Raises:
-        KeyError: A folder given in the config does not exist
-
-    Returns:
-        tuple: created folders: (isparrow-homefolder, modelsfolder, datafolder, outputfolder, examplefolder)
-    """
-    if "home" not in base_cfg_dirs:
-        raise KeyError("The home folder for iSparrow must be given in the base config")
-
-    if "data" not in base_cfg_dirs:
-        raise KeyError("The data folder for iSparrow must be given in the base config")
-
-    if "output" not in base_cfg_dirs:
-        raise KeyError(
-            "The output folder for iSparrow must be given in the base config"
-        )
-
-    ish = Path(base_cfg_dirs["home"]).expanduser().resolve()
-    isd = Path(base_cfg_dirs["data"]).expanduser().resolve()
-    iso = Path(base_cfg_dirs["output"]).expanduser().resolve()
-
-    for p in [ish, isd, iso]:
-        p.mkdir(parents=True, exist_ok=True)
-
-    return ish, isd, iso
+CONFIG = None
 
 
 # add a fixture with session scope that emulates the result of a later to-be-implemented-install-routine
-@pytest.fixture(scope="session", autouse=True)
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
have a new install for each tested module is a bit better a compromise between redoing work and possible interactions than allowing session wide test interactions.